### PR TITLE
Rtf 1398

### PR DESCRIPTION
**Pull request by klaudiaknafel**
Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework/blob/master/CHANGELOG.md)

### CODE REVIEW

File: source/Relativity.Testing.Framework/Models/Group/Group.cs [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
+using System.Collections.Generic;
+
+namespace Relativity.Testing.Framework.Models
 {
 	/// <summary>
 	/// Represents the Relativity group object.
 	/// </summary>
-	public class Group : NamedArtifact, IFillsRequiredProperties<Group>
-	{
+	public class Group : TimeStampedNamedArtifact, IFillsRequiredProperties<Group>, IHaveGuids
```
Comment (briantoms) [inline source/Relativity.Testing.Framework/Models/Group/Group.cs:9]: Can I just say I love the interface name `IHaveGuids`?

**Review by briantoms — DISMISSED** [review]

File: source/Relativity.Testing.Framework/Models/Group/Group.cs (hunk 1 above)
Comment (klaudiaknafel) [reply source/Relativity.Testing.Framework/Models/Group/Group.cs:9]: I think that's @jonatanjakoniuk doing

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.3.0] - 2021-09-27
+
+### Added
+
+- Properties for Group: Guids, Actions, Meta, CreatedBy, CreatedOn, LastModifiedBy, LastModifiedOn.
```
Comment (paulletang) [inline CHANGELOG.md:12]: With these properties on the models now, the fields we try to send to the API request will change.
Will that cause any problems?

I'm specifically thinking about the readonly ones: CreatedBy, CreatedOn, LastModifiedBy, LastModifiedOn

Comment (klaudiaknafel) [reply CHANGELOG.md:12]: They will not be sent in any of the requests (just will be included in the responses). Create and update requests (DTOs) don't have these properties

**Review by briantoms — APPROVED** [review]

**Review by paulletang — APPROVED** [review]

**Pull request merged**